versioned_docs/version-v0.15/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -435,6 +435,20 @@ Events of pull requests.
 
 ### Domain 4 - CI/CD(WIP)
 
+#### cicd_scopes

```

Comment:
Please replicate the changs to the v0.15 doc as well.

Reply:
fixed